tools/reviewer/web/src/app/dashboard/index.ts

```diff
@@ -14,7 +14,7 @@ import {
   ReviewDiscussionComponent,
   ReviewFilesComponent,
   ReviewsPanelComponent,
-  ReviewTitlebarComponent
+  ReviewTitlebarComponent,
```

Comment:
Can we fix the trailing commas so that it's consistent in the whole app. I don't care if are enforcing trailing comma or enforcing no-trailing comma. But it has to be consistent in the whole app at any given point in time.

Reply:
Yeah, let's do that.

Reply:
It's great that you agree.
Do you know of a tool that can automatically check this?

Reply:
Yeah, we are using linter already.